### DISCUSSION TITLE
Chaining data attribute selections that use the same style?

### DISCUSSION THREAD

**Discussion by binaryartifex** [tailwindlabs/tailwindcss#11806] [Help]
Use case -> im using React Aria Components and am styling a checkbox. it has data attributes for 'selected' and 'indeterminate', both of which id like to have the same styling. I sent a prayer to the tailwind gods hoping this would work...alas, twas not so....is it even possible?

rule applied to the label wrapping the checkbox
`"group"`

rule applied to the svg within the checkbox
`"group-data-[hovered][selected]:border-primary-500",`

**Comment by wongjn** [comment]
Consider using an [arbitrary group variant](https://tailwindcss.com/docs/hover-focus-and-other-states#arbitrary-groups) in conjunction with [`:is()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) like:

```html
<div class="group-[:is([data-selected],[data-indeterminate])]:border-primary-500">
```

See a live example in [this Tailwind Play](https://play.tailwindcss.com/uVWS1hY8fG).

**Comment by binaryartifex** [comment]
sweet baby jesus its beautiful. you sir, are a gentleman and a scholar.

**Comment by IvanTorson** [comment]
I think the Gods amount us !

**Comment by binaryartifex** [comment]
@wongjn im trying to create an arbitrary rule to style a button on the condition that it is disabled, however the parent (group) component is not disabled. use-case being a number input with up and down arrows. when min/max is reached, the button itself has reduced opacity. however if the input as a whole is disabled, and one of the inner buttons is disabled, i don't want that disable stying to be applied... is this possible?

**Reply by wongjn** [reply]
Yes, I think it would be possible.

**Comment by binaryartifex** [comment]
truth be told in retrospect i was just being overly pedantic. but now i gotta know if its possible in principle...
